server/app/views/questiontypes/TextQuestionFragment.html

```diff
@@ -0,0 +1,17 @@
+<div th:fragment="text (question)"
+     th:with="textQuestion=${question.createTextQuestion()}, questionId=${#strings.randomAlphanumeric(8)}">
+    <!-- Title and Help Text -->
+    <div th:replace="~{questiontypes/QuestionBaseFragment :: titleAndHelpText(${question}, ${questionId})}"></div>
```

Comment:
question: Should this be a label or should there be a label somewhere?

Reply:
This div gets replaced by the contents of that template. Right now that template uses a <legend> but I think I need to do a small refactor to make it conditionally a label or legend depending on whether it has one or multiple inputs. I was thinking of making that its own PR, but I could do it here if you prefer.

Reply:
Separate is fine